lib/safe-pg-migrations/plugins/statement_insurer/add_column.rb

```diff
@@ -53,11 +53,19 @@ def backfill_column_default(table_name, column_name)
 
         Helpers::Logger.say_method_call(:backfill_column_default, table_name, column_name)
 
-        Helpers::BatchOver.new(model).each_batch do |batch|
+        batch_handler = lambda do |batch|
           batch.update_all("#{quoted_column_name} = DEFAULT")
 
           sleep SafePgMigrations.config.backfill_pause
         end
+
+        backfill_batch_size = SafePgMigrations.config.backfill_batch_size
+
+        if ActiveRecord.version >= Gem::Version.new('7.1')
```

Comment:
until the patch on rails is not provided/approved/merged/released, i think is to early to switch towards rails implementation. WDYT ?

Reply:
Agreed, let's keep this PR in draft and keep the new small optim from https://github.com/doctolib/safe-pg-migrations/pull/138.

Reply:
I've just pushed a proposal:

* https://github.com/rails/rails/issues/51242
* https://github.com/rails/rails/pull/51243

Let's see if it gets some traction 🤞.